c_glib/test/test-array.rb

```diff
@@ -116,6 +116,18 @@ def test_to_s
     CONTENT
   end
 
+  def test_validation_success
+    array = build_int32_array([1, 2, 3, 4, 5])
+    assert_equal(true, array.validate)
+  end
+
+  def test_validation_fail
+    array = Arrow::Int8Array.new(-1, Arrow::Buffer.new(""), Arrow::Buffer.new(""), -1)
+    assert_raise(Arrow::Error::Invalid) do
+      array.validate
+    end
+  end
+
```

Comment:
Could you move this to the end of this file to align with `basic-array.h` order?

Could you use `sub_test_case` when we have multiple patterns for a method?

```ruby
sub_test_case("#validate") do
  def test_valid
  end

  def test_invalid
  end
end
```

Reply:
Thanks fixed.